app/controllers/finance/npq/course_payment_breakdowns_controller.rb

```diff
@@ -4,14 +4,23 @@ module Finance
   module NPQ
     class CoursePaymentBreakdownsController < BaseController
       def show
-        @statement         = Finance::Statement::NPQ.find_by_name(params[:statement_id])
         @npq_lead_provider = lead_provider_scope.find(params[:lead_provider_id])
+        @cpd_lead_provider = @npq_lead_provider.cpd_lead_provider
+
+        @statement = Finance::Statement::NPQ.find_by(
+          name: params[:statement_id],
+          cpd_lead_provider: @cpd_lead_provider,
+        )
+
+        if @statement.name == "January 2022"
```

Comment:
yea kind of a rush job here. it essentially makes that particular statement "anonymous". Meaning it won't look for declarations assigned to the statement. Instead it will simply mop up and look for declarations not associated with any statements.

I think once we get into the flow these things will iron out such as adding in new statements and making the most recent one display a "current" view

Reply:
Addressed in separate PR